server/routes/checkAnswers.js

```diff
@@ -47,3 +140,47 @@ module.exports = function Index({ formService, authenticationMiddleware }) {
 
   return router
 }
+
+const getRestraintPositions = positions => {
+  if (Array.isArray(positions)) {
+    return positions.join(', ')
+  }
+  return ''
+}
+
+const convertArrayOfObjectsToString = (dataArray = []) => {
```

Comment:
maybe rename this to `extractNames` ?

Reply:
I've used that function for 'Body worn cameras' too so I'm not sure about changing it to something related just to names?

Reply:
it extracts a field called name, so that suggests that the data is being incorrectly stored.

Reply:
I thought extract() was not descriptive enough so have renamed to convertArrayOfObjectsToStringUsingSpecifiedKey() instead. 
The refactored function now takes another arg that is the end of the name property for the element, i.e. cameraNum, name.